x-pack/plugins/observability_solution/infra/server/routes/infra/lib/constants.ts

```diff
@@ -6,6 +6,8 @@
  */
 
 export const BUCKET_KEY = 'host.name';
+export const EVENT_MODULE = 'event.module';
+export const METRICSET_MODULE = 'metricset.module';
```

Comment:
Perhaps these could be in the [constants](https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability_solution/infra/common/constants.ts) in `common` instead. It will allow the client to also access them.

Reply:
Good point, thanks ✅